ansible/roles/hana_actions/tasks/main.yml

```diff
@@ -0,0 +1,164 @@
+---
+- name: Prepare HANA Action - Dump cluster status (crm status)
+  command: crm status
+  become: true
+  register: crm_initial
+  changed_when: false
+
+- name: Prepare HANA Action - Assert this node is currently MASTER
+  assert:
+    that:
+      - crm_initial.stdout is search('\* Masters:\s*\[\s*{{ inventory_hostname }}\s*\]')
+    fail_msg: "{{ inventory_hostname }} is NOT the master node – aborting test"
+    success_msg: "{{ inventory_hostname }} is the master node"
+
+- name: Prepare HANA Action - Wait cluster idle
+  command: cs_wait_for_idle --sleep 5
+  register: idle
+  retries: "{{ cs_wait_timeout // 5 }}"
+  delay: 5
+  become: true
+  until: idle.rc == 0
+
+# SBD related stuff
+- name: Prepare HANA Action - Configure SBD start-delay
```

Comment:
SBD delay tasks:
- it  is so complex and involving for the SUT
- all of the tasks are under the relatively complex condition `when: action == 'crash' or (action == 'stop' and cloud_platform_name == 'EC2')`


I propose to move it under a dedicated role, all the tasks in th erole are without the `when` condition, we only have the condition once when we decide if to call the role